packages/nameguard-react/src/components/DisplayedName/DisplayedName.tsx

```diff
@@ -30,9 +32,10 @@ export function DisplayedName({
   return (
     <TruncatedText
       text={displayName}
-      textStylingClasses={textStylingClasses + " ens-webfont"}
       maxTooltipWidth={maxTooltipWidth}
       maxDisplayWidth={maxDisplayWidth}
+      textStylingClasses={textStylingClasses + " ens-webfont"}
+      tooltipContentStylingClasses={tooltipContentStylingClasses}
```

Comment:
Shouldn't this also add "ens-webfont"?

Reply:
Only if the user wants to, right?

Reply:
@FrancoAguzzi   It should ALWAYS happen inside `DisplayedName`!